contrib/cirrus/setup_environment.sh

```diff
@@ -100,7 +100,13 @@ case "$CG_FS_TYPE" in
 esac
 
 # Force the requested database backend without having to use command-line args
+# As of #20318 (2023-10-10) sqlite is the default, but for complicated reasons
+# we still (2023-11-01) have to explicitly create a containers.conf. See
+# comments in #20559.
+# FIXME: some day, when new CI VMs are in place with podman >= 4.8 installed
+# from RPM, un-comment the 'if' below. That will confirm that sqlite is default.
```

Comment:
Oh, interesting idea. What I don't like is that it requires invoking podman, which is exactly what I'd like to avoid (I've been struggling to find a way to remove the call in `remove_packaged_podman()`). I also don't fully trust that the logic will work, and am reluctant to leave that as a problem for the next person bumping VMs.